recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - netcdf4
     - matplotlib-base
     - pybtex
-    - statsmodels
+    - statsmodels ==0.14.0
```

Comment:
@mkstratos I see this is the pin in evv4esm so what needs to be done here, but you might want to consider un-restricting the upper bound (`>=` instead) upstream. Here's a good analysis of why:
https://iscinumpy.dev/post/bound-version-constraints/